docs/guide/built-in-rules-callback.md

```diff
@@ -26,14 +26,14 @@ function (mixed $value, Callback $rule, ValidationContext $context): Result;
 
 where:
 
-- `$value` is validated value;
-- `$rule` is a reference to original `Callback` rule;
+- `$value` is the validated value;
+- `$rule` is a reference to the original `Callback` rule;
 - `$context` is a validation context;
-- returned value is a validation result instance with errors or without them.
+- Returned value is a validation result instance with errors or without errors.
```

Comment:
Why repeating "errors" if nothing else is mentioned?

Reply:
Came up with a better wording.